test/include/chroot_env.sh

```diff
@@ -76,13 +76,13 @@ teardown_chroot_env()
         umount "${target_root_dir}/proc"
     fi
 
-    mounts="${overlayfs_dir}/rom ${overlayfs_dir}"
+    mounts="${overlayfs_dir}/rom ${overlayfs_dir} ${target_root_dir}"
     for mount in ${mounts}; do
         if grep -q "${mount}" "/proc/mounts"; then
             umount "${mount}"
         fi
-        if [ -d "${mount}" ] && [ -z "${mount##*overlayfs_*}" ]; then
-            rm -r "${mount:?}"
+        if [ -d "${mount}" ] && [ -z "${mount##*overlayfs*}" ]; then
```

Comment:
p.s. you could also put your template/mask in a variable and reference it in the needed spots :p (just a tip, :tipping_hand_man:)

Reply:
ditto, per EMP-406